### PR TITLE
adding unit tests for utils

### PR DESCRIPTION
**Pull request by microcat49**

### CODE REVIEW

File: src/test/utils/validators.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+      expect(isPositiveNumber("4.67")).toBeTruthy();
+      expect(isPositiveNumber('.98')).toBeTruthy();
+      expect(isPositiveNumber("0.333")).toBeTruthy();
+      expect(isPositiveNumber("56.8")).toBeTruthy();
+      expect(isPositiveNumber("6")).toBeTruthy();
+
+});
+
+test('isPositiveWholeNumber()', () => {
```
[the remark this answers is not in the document]
Comment (microcat49) [reply src/test/utils/validators.test.js]: Hey @knod so I looked at isPositiveWholeNumber() it doesn't take scientific notation?

[the remark this answers is not in the document]
Comment (knod) [reply src/test/utils/validators.test.js]: @microcat49: Exactly - it should make sure that scientific notation is invalid. We should make sure that it does that properly.

[the remark this answers is not in the document]
Comment (microcat49) [reply src/test/utils/validators.test.js]: @knod Oh okay I see I can make the changes and resubmit the pull request?

**Comment by microcat49** [comment]
Hey @knod I accidentally merged this pull request. I don't think I can reverse it. Can you do it for me?

**Comment by knod** [comment]
@microcat49: Don't worry about it, just do another PR building on that one.

**Pull request merged**